lib/rules/validate-jsdoc/require-description-complete-sentence.js

```diff
@@ -0,0 +1,62 @@
+module.exports = requireDescriptionCompleteSentence;
+module.exports.scopes = ['function'];
```

Comment:
What do I put there for scope of a variable?

is it just 'variable' ?

I could not find a single rule that uses that and instead uses file.